src/integration-test/tests/citizen/endToEnd/full_admission_test.ts

```diff
@@ -11,17 +11,48 @@ import {
   ClaimantResponseTestData,
   UnreasonableClaimantResponseTestData
 } from 'integration-test/tests/citizen/claimantResponse/data/ClaimantResponseTestData'
+import { createClaimData } from '../../../data/test-data'
```

Comment:
please replace with correct path: example in the line above integration-test/tests/citizen